src/main/resources/templates/share/packings.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="ja">
+<th:block th:replace="~{fragment/head :: head(持ち物リスト)}"/>
+<body class="flex flex-col min-h-screen">
+<th:block th:replace="~{fragment/header :: header}"/>
+<main class="flex flex-col items-center flex-grow">
+    <h2 class="text-center sm:text-3xl text-2xl font-medium mt-10">持ち物リスト</h2>
+    <form id="item-form" action="/share/packings" method="POST" class="w-full sm:max-w-lg max-w-xs m-24">
+        <input type="hidden" th:name="${_csrf.parameterName}" th:value="${_csrf.token}" />
```

Comment:
```suggestion
    <form id="item-form" th:action="@{/share/packings}" method="POST" class="w-full sm:max-w-lg max-w-xs m-24">
```
`th:action="@{}"` の形にすれば、自動でCSRFトークン入ったinputが生成されるから
手動で追加しなくても大丈夫かも

Reply:
修正した
[833dd33](https://github.com/saitokazumasa/tabisketch/pull/374/commits/833dd3370f27040b5ea8ee486a3602e66611f858)